lib/Auth/Process/PrivacyideaAuthProc.php

```diff
@@ -204,6 +242,22 @@ private function enrollToken($stateID, $username)
         return "";
     }
 
+    /**
+     * @return void
+     * @throws Exception
+     */
+    public static function handleLogout()
+    {
+        SimpleSAML_Logger::debug("privacyIDEA: handle logout. Remove SSO data.");
+
+        /*
+         * This method is static and called after login without providing state
+         * and we can't implement separate SSO for different IdP, SP etc.
+         * So we delete single SSO data.
+         */
+        Session::getSessionFromRequest()->deleteData('privacyidea:privacyidea:sso', 'data');
```

Comment:
why is this not SimpleSAML_Session?

Reply:
Yeah, your right. Changed in commit #c364017.

Btw. interesting, that this time was no error...